envisage/ui/tasks/tasks_application.py

```diff
@@ -60,7 +61,7 @@ class TasksApplication(Application):
     active_window = Instance("envisage.ui.tasks.task_window.TaskWindow")
 
     # The Pyface GUI for the application.
-    gui = Instance("pyface.gui.GUI")
+    gui = Supports("pyface.i_gui.IGUI")
```

Comment:
Is there a use-case for allowing adaptation here? If not, I'd rather go with a simple `Instance`.

Reply:
I was emulating what we already have in envisage `GUIApplication` - https://github.com/enthought/envisage/blob/8eddfc402f2be38bf8774e48ab31a118d8dcb565/envisage/ui/gui_application.py#L43-L44.

Updating to use `Instance` instead of `Supports`.